lib/pinchflat_web/controllers/pages/page_html/history_table_live.ex

```diff
@@ -97,13 +98,20 @@ defmodule Pinchflat.Pages.HistoryTableLive do
     |> Repo.preload(:source)
   end
 
-  defp generate_base_query do
+  defp generate_base_query("pending") do
     MediaQuery.new()
     |> MediaQuery.require_assoc(:media_profile)
     |> where(^dynamic(^MediaQuery.downloaded() or ^MediaQuery.pending()))
```

Comment:
Since this is just for `pending` media items, could you please remove the `MediaQuery.downloaded()` from this query?

Reply:
ah shoot, forgot to change it. First i was thinking about a "All" tab. Then i realized there are also the "other" entries.